### DISCUSSION TITLE
Filter with 2 attributes

### DISCUSSION THREAD

**Discussion by iweezagency** [meilisearch/product#599] [Feedback & Feature Proposal]
Hello,

I looking a way to filter results as in SQL = WHERE col1 <= col2,

In my case I use meilisearch to search with geoRadius and I have a "item_range" attribute I want to do

$options['filter'] = "_geoDistance <= item_range AND _geoRadius({$lat}, {$lng}, {$range})";

But I have no result, it is exist a way to deal with it ?

thanks

**Comment by gmourier** [comment]
Hi @iweezagency 👋

It is not possible to filter on `_geoDistance`, the field is calculated at run time for information display.

Can you tell me more about your use-case for using `_geoDistance` in your filter expression? Maybe we can find a workaround.

Thanks!

**Reply by iweezagency** [reply]
Thanks for your answer @gmourier, 

In fact we have a list of places, each place have to define their own range.
